### ISSUE TITLE
replicating conditions 5 times

### ISSUE THREAD

**Issue by umairafzal9245** [LTH14/fractalgen#22]
Were 5 conditions intentionally used at fractal level 0, or is it a bug?
In this loop: cond_list = [class_embedding for _ in range(5)]
File_Name: models/fractalgen.py

**Comment by LTH14** [comment]
This is just to make it consistent as the later levels using 5 conditions.

**Comment by umairafzal92786** [comment]
@LTH14 why the index for r,g,b logits start from 0 instead of 1? at 0 we are concatenating a condition.

**Comment by umairafzal92786** [comment]
what is the purpose of passing 5 conditions because on next level in loss network we are just using cond_list[0]

**Comment by LTH14** [comment]
num_conds will specify how many conditions to use
